docs/early-access-program.mdx

```diff
@@ -3,109 +3,32 @@ title: Pieces Early Access Program
-- "Take the function from example_function.dart and add it as a method to the class in example_class.dart"
-- "What did Mark say about requests to the xyz api in slack?"
-
-## Data and Privacy
-**Your workstream data is captured and stored locally on-device.** At no point will anyone, including the Pieces team, have access to this data unless you choose to share it with us.
-
-The Workstream Pattern Engine triangulates and leverages on-task and technical context across developer-specific tools you're actively using. The bulk of the processing that occurs within the Workstream Pattern Engine is filtering, which utilizes our on-device machine learning engines to ignore sensitive information and secrets. This enables the highest levels of performance, security, and privacy.
-
-Lastly, for some advanced components within the Workstream Pattern Engine, blended processing is required to be set via user preferences and you will need to leverage a cloud-powered Large Language Model as your copilot’s runtime.
-
-That said, you can leverage Local Large Language Models, but this may reduce the fidelity of output and requires a fairly new machine (2021 and newer) and ideally a dedicated GPU for this. You can read [this blog](https://code.pieces.app/blog/how-to-run-an-llm-locally-with-pieces) for more information about running local models on your machine.
+## Goals of the Early Access Program
+Our main priority for this program is to test our latest features in as many development environments and tooling configurations as possible. From you, we want to hear pain points, any confusion you have regarding how features work, if you feel they add value to your daily workflow, and **how we can make them more valuable**.
 
-As always, we've built Pieces from the ground up to put you in control. With that, **the Workstream Pattern Engine may be paused and resumed at any time**. Additionally, users may also clear locally persisted data and opt out of the Workstream Pattern Engine Early Access Program with a simple request to the team. Details on how to pause or opt-out may be found below.
+## How to Join
+If you are not currently a part of the Pieces for Developers Early Access Program, you can sign up within the Pieces Desktop App. Ensure you are signed in, then either:
+- Navigate to your `Account & Cloud Integrations` settings, or
+- Press `⌘/CTRL + Shift + P` to open the Pieces Power Menu and type `Early Access Program`.
 
-## Goals of the Early Access Program
-Our main priority for this program is to test the workstream pattern engine in as many development environments and tooling configurations as possible. From you, we want to hear pain points, any confusion you have regarding how the feature works, if you feel it adds value to your daily workflow, and **how we can make it more valuable**.
+Once you have joined, please return to this page to review how to get started with the latest features.
```

Comment:
Screenshot of account profile with EAP badge i.e. "Once you're in you'll see this Verified EAP User Badge..."

Reply:
Related to #427